pos_margin/static/src/js/screens.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2021 - Today: GRAP (http://www.grap.coop)
+// @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+odoo.define("pos_margin.screens", function (require) {
+    "use strict";
+
+    var screens = require("point_of_sale.screens");
+
+    screens.OrderWidget.include({
+        update_summary: function () {
+            this._super.apply(this, arguments);
+            var order = this.pos.get_order();
+            if (!order.get_orderlines().length) {
+                return;
```

Comment:
I haven't tested, but isn't it necessary to clear the summary in case you remove all order lines?

Reply:
no, the summary is hidden if nolines.

length === 0

![image](https://user-images.githubusercontent.com/3407482/121523048-10725c00-c9f6-11eb-9302-55060221005d.png)

length !== 0

![image](https://user-images.githubusercontent.com/3407482/121523126-2849e000-c9f6-11eb-8b76-e39cb1e67eda.png)